src/js/utils.mjs

```diff
@@ -32,7 +36,9 @@ export function getParam(param) {
 export function getCartCount(){
   const cartItems = getLocalStorage("so-cart");
   let count = 0;
-  cartItems.forEach((item) => count += item.Quantity);
+  if(cartItems){
```

Comment:
I noticed an error in the console if the cart was empty and this was the fix for it.

Reply:
This works fine. If you really want to shorten it, we could use the same approach in the addProductToCart function with the nullish operator (??):
   const cartItems = getLocalStorage("so-cart") ?? [];

Honestly, either way would work just as well.